.changes/unreleased/Dependencies-20230728-135227.yaml

```
@@ -0,0 +1,6 @@
+kind: Dependencies
+body: Support dbt-semantic-interfaces 0.2.0
+time: 2023-07-28T13:52:27.207241-07:00
+custom:
+  Author: QMalcolm
+  PR: "8247"
```

This type and Security specifically point at PR instead of issue.
```suggestion
  PR: "8250"
```